assets/cubyz/biomes/prairie.zig.zon

```diff
@@ -0,0 +1,128 @@
+.{
+	.properties = .{
+		.dry
+	},
+	.minHeight = 40,
+	.maxHeight = 44,
+	.minRadius = 100,
+	.maxRadius = 160,
+	.roughness = 1,
+	.hills = 3,
+	
+	.music = "cubyz:sunrise",
+
+	.validPlayerSpawn = false,
```

Comment:
It has trees, so I think it should be a valid spawn point.
This variable is to prevent soft-locking the player in an area with insufficient resources to get started.
I think the player should be allowed to spawn in as many biomes as possible, so there is more variety when creating a new world.